src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -57,11 +61,13 @@ export default {
     defaultOpen: { control: "boolean" },
     showArrow: { control: "boolean" },
     side: { control: "select", options: ["top", "right", "left", "bottom"] },
+    type: { control: "inline-radio", options: ["text", "button"] },
   },
 };
 
 export const Playground = {
   args: {
     side: "bottom",
+    type: "text",
```

Comment:
in the story let's have only properties that belong to the component to avoid confusion for the users